build_goreleaser/lookoutv2/Dockerfile

```diff
@@ -1,10 +1,27 @@
+ARG NODE_BUILD_IMAGE=node:16.14-buster
```

Comment:
I've copied the additions from https://github.com/armadaproject/armada/commit/51262cd10db51b484d3ec395ba45529a6a2e9303#diff-ae140648c352a82c0b0d88cad81f407a775f5503109fc9d8b6c4d236bda5717c.